julia_src/reopt_model.jl

```diff
@@ -648,6 +663,68 @@ function add_load_balance_constraints(m, p)
 end
 
 
+function add_load_balance_constraints_offgrid(m, p)
+	##Constraint (8b): Electrical Load Balancing without Grid
+	@constraint(m, ElecLoadBalanceNoGridCon[ts in p.TimeStep],
+		sum(p.ProductionFactor[t,ts] * p.LevelizationFactor[t] * m[:dvRatedProduction][t,ts] for t in p.ElectricTechs) +
+		sum( m[:dvDischargeFromStorage][b,ts] for b in p.ElecStorage )  ==
+		sum( sum(m[:dvProductionToStorage][b,t,ts] for b in p.ElecStorage) +
+			 m[:dvProductionToCurtail][t,ts]
+		for t in p.ElectricTechs) +
+        sum(m[:dvThermalProduction][t,ts] for t in p.ElectricChillers )/ p.ElectricChillerCOP +
+        sum(m[:dvThermalProduction][t,ts] for t in p.AbsorptionChillers )/ p.AbsorptionChillerElecCOP +
+		(p.ElecLoad[ts] * m[:dvLoadServed][ts])
```

Comment:
we could fix `dvLoadServed` to one in "on grid" analyses and re-use the existing load balance constraint?